js/App.js

```diff
@@ -12,6 +13,9 @@ class App extends React.Component {
 				<p>
 					You have been on my site for <Counter start={0}/> seconds
 				</p>
+				<p>
+					You only have <Countdown start={10}/> seconds!!
```

Comment:
Try defining a function somewhere in this component, and passing that to countdown, and call that instead of the alert.